### PR TITLE
Update visual-studio to use VisualStudioForMac.app in version 7.0.0.3146

### PR DESCRIPTION
**Pull request by jacano**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by commitay** [comment]
This cask requires `mono-mdk` 5.0. Updated with https://github.com/caskroom/homebrew-cask/pull/33895

Please add `depends_on cask: 'mono-mdk'`

visual studio

File: Casks/visual-studio.rb [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+  url "https://dl.xamarin.com/VsMac/VisualStudioForMac-#{version}.dmg"
+  appcast 'http://xamarin.com/installer_assets/v3/vsmac/Mac/Universal/InstallationManifest.xml',
+          checkpoint: '54618531df092f73e61a1658cfdaf83081edebaeb924a8d0d46ab022b46175dd'
   name 'Visual Studio for Mac'
   homepage 'https://www.visualstudio.com/vs/visual-studio-mac/'
 
-  installer manual: 'Install Visual Studio for Mac Preview.app'
+  app 'Visual Studio.app'
 
   uninstall delete: '/Applications/Visual Studio.app'
```
Comment (commitay) [inline Casks/visual-studio.rb]: `uninstall delete: '/Applications/Visual Studio.app'` can be removed.

**Comment by miccal** [comment]
Thank you for your contribution.

**Comment by jacano** [comment]
Yes, didn't notice mono-mdk was a dep because I had it already installed :disappointed_relieved:
Nice catch!!
Seems like @miccal already did it 👍 

Btw guys, congrats for such a great project, I really love it!!

**Pull request merged**